README.md

```diff
@@ -94,6 +94,8 @@ export default [
         createTypeScriptImportResolver({
           alwaysTryTypes: true, // always try to resolve types under `<root>@types` directory even it doesn't contain any source code, like `@types/unist`
 
+          bun: true, // resolve Bun modules, defaults to false
```

Comment:
There is no need to set this option for `bun --bun` or [`run#bun`](https://bun.sh/docs/runtime/bunfig#run-bun-auto-alias-node-to-bun) enabled.

Reply:
yeah, I guess I'm thinking that there will be a lot of projects that don't want to switch to running ESLint with Bun (eg. just introducing some small Bun scripts in a larger project)

but happy to also include this - do you think that it would be enough to mention it in the comment? or should it have a paragraph outside of the code block?

Reply:
I think a new paragraph for `Bun` is better.

Reply:
I added "Other environments" and "Bun" headings, and instructions for the three methods described in [`eslint-import-resolver-typescript@4.2.0`](https://github.com/import-js/eslint-import-resolver-typescript/releases/tag/v4.2.0):

![Screenshot 2025-03-25 at 17 20 54](https://github.com/user-attachments/assets/ea8aafab-7932-417b-83c3-f205de58cfd2)

Commits:

- c0e3ce25b92fcfb68108309acd35e8964e499511
- 29e410e97794312a7b9e07e6d599e4d59dec3837

Thoughts?

Reply:
`default: false` seems like won't try `bun` at all without this option, I think we should only reference to the new section instead.

Reply:
added a link to the comment and removed the mention of the default:

- d86965d3a6f18a492a95b6d4cb7d1171984d94a8

I think it's good to have configuration values in the "Configuration" top-level code blocks and not somewhere else